libr/anal/fcn.c

```diff
@@ -1077,7 +1077,7 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			if (anal->cur) {
 				/* if UJMP is in .plt section just skip it */
 				RBinSection *s = anal->binb.get_vsect_at (anal->binb.bin, addr);
-				if (s && s->name) {
+				if (s && strlen(s->name)) {
```

Comment:
just use s->name[0]. no need to count all the chars of a string to know if its != 0